samples/test/util.py

```diff
@@ -79,12 +79,6 @@ def test_wrapper(
                     case.pipeline_func._component_human_name))
                 continue
 
-            # TODO(chensun): renable v2 engine tests
```

Comment:
If I'm not mistaken, you still disabled all the v2 engine test from individual test files, right?
I think you can skip all the test changes in this PR and focus on only the v2 code, while https://github.com/kubeflow/pipelines/pull/6932 tries to bring the samples compatible with v2 and enable as mush v2_engine tests as possible. WDYT?

Reply:
Thanks, I'll keep only the v2 engine changes in this PR.